File: Arrays/Easy/Move_zeros.java
public class Move_zeros{
    public static void main(String[] args){
        int[] nums = {0, 1, 0, 3, 12};
        int n = nums.length;
        int count = 0;
        for(int i = 0; i < n; i++){
            if(nums[i] != 0){
                nums[count++] = nums[i];
            }
        }
        while(count < n){
            nums[count++] = 0;
        }
        System.out.println("Array after moving zeros to the end: ");
        for(int i = 0; i < n; i++){
            System.out.print(nums[i] + " ");
        }
    }
}

File: Arrays/Easy/Python/Move_zeros.py
def move_zeros(nums):
    n = len(nums)

    count = 0

    for num in nums:
        if num != 0:
            nums[count] = num
            count += 1
    
    while count < n:
        nums[count] = 0
        count += 1
        


nums = [0, 1, 0, 3, 12]
move_zeros(nums)
print(nums)